test/e2e/multihoming.go

```diff
@@ -1169,7 +1169,7 @@ var _ = Describe("Multi Homing", func() {
 					),
 				),
 				ginkgo.Entry(
-					"for a routed topology when the multi-net policy describes the allow-list using pod selectors",
+					"using pod selectors for a routed topology",
```

Comment:
for my own understanding: what's a routed topology in this context?

Reply:
I think that means Layer3, like that one with the router :D

Reply:
ah gotcha thanks